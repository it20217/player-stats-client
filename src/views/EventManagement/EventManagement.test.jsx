import { MemoryRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import EventManagement from './EventManagement';

describe("EventManagement component",() => {
  it("renders correctly", () => {
    const component = render(
      <MemoryRouter initialEntries={["/events"]}>
        <EventManagement/>
      </MemoryRouter>
    );
    expect(component).toMatchSnapshot();
  });
});