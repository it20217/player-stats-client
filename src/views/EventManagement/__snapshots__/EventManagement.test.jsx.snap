// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventManagement component renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="py-12 min-h-screen"
      >
        <p
          class="text-left ml-8 text-base sm:text-lg md:text-xl lg:text-2xl font-bold leading-normal text-gray-800"
        >
          Sport events
        </p>
        <div
          class="mx-auto container bg-white dark:bg-gray-800 shadow rounded"
        >
          <div
            class="flex flex-col lg:flex-row p-8 justify-between items-start lg:items-stretch w-full"
          >
            <div
              class="w-full lg:w-1/4 xl:w-1/3 flex flex-col lg:flex-row items-start lg:items-center"
            >
              <div
                class="w-full relative mb-2 lg:mb-0 lg:mr-4"
              >
                <div
                  class="absolute text-gray-600 dark:text-gray-400 flex items-center pl-4 h-full"
                >
                  <svg
                    class="icon icon-tabler icon-tabler-search"
                    fill="none"
                    height="18"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="1.5"
                    viewBox="0 0 24 24"
                    width="18"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      stroke="none"
                    />
                    <circle
                      cx="10"
                      cy="10"
                      r="7"
                    />
                    <line
                      x1="21"
                      x2="15"
                      y1="21"
                      y2="15"
                    />
                  </svg>
                </div>
                <label
                  class="hidden"
                  for="search"
                />
                <input
                  class="bg-gray-100 dark:bg-gray-700 text-gray-600 dark:text-gray-400 focus:outline-none focus:border focus:border-indigo-700 font-normal w-full h-10 flex items-center pl-12 text-sm border-gray-300 dark:border-gray-200 rounded border"
                  id="search"
                  placeholder="Search"
                />
              </div>
            </div>
            <div
              class="w-full lg:w-3/4 xl:w-2/3 flex flex-col lg:flex-row items-start lg:items-center justify-between"
            >
              <div
                class="relative w-full lg:w-1/4 my-2 lg:my-0 lg:mx-2 xl:mx-4 z-10"
              >
                <div
                  class="absolute z-0 inset-0 m-auto mr-2 xl:mr-4 z-0 w-5 h-5"
                >
                  <svg
                    class="icon cursor-pointer icon-tabler icon-tabler-chevron-down"
                    fill="none"
                    height="20"
                    stroke="#a0aec0"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="1.5"
                    viewBox="0 0 24 24"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      stroke="none"
                    />
                    <polyline
                      points="6 9 12 15 18 9"
                    />
                  </svg>
                </div>
                <select
                  aria-label="Selected tab"
                  class="relative z-10 cursor-pointer focus:outline-none focus:border-gray-800 focus:shadow-outline-gray text-sm form-select block w-full py-2 px-2 xl:px-3 border border-gray-300 dark:border-gray-200 rounded text-gray-600 dark:text-gray-400 appearance-none bg-transparent"
                  disabled=""
                >
                  <option
                    class="text-sm text-gray-600 dark:text-gray-400"
                  >
                    Representatives
                  </option>
                  <option
                    class="text-sm text-gray-600 dark:text-gray-400"
                  >
                    Representatives
                  </option>
                  <option
                    class="text-sm text-gray-600 dark:text-gray-400"
                  >
                    Representatives
                  </option>
                  <option
                    class="text-sm text-gray-600 dark:text-gray-400"
                  >
                    Representatives
                  </option>
                  <option
                    class="text-sm text-gray-600 dark:text-gray-400"
                  >
                    Representatives
                  </option>
                </select>
              </div>
              <div
                class="relative w-full lg:w-1/4 my-2 lg:my-0 lg:mx-2 xl:mx-4 z-10"
              >
                <div
                  class="z-0 absolute inset-0 m-auto mr-2 xl:mr-4 z-0 w-5 h-5"
                >
                  <svg
                    class="icon cursor-pointer icon-tabler icon-tabler-chevron-down"
                    fill="none"
                    height="20"
                    stroke="#a0aec0"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="1.5"
                    viewBox="0 0 24 24"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      stroke="none"
                    />
                    <polyline
                      points="6 9 12 15 18 9"
                    />
                  </svg>
                </div>
                <select
                  aria-label="Selected tab"
                  class="relative z-10 cursor-pointer focus:outline-none focus:border-gray-800 focus:shadow-outline-gray text-sm form-select block w-full py-2 px-2 xl:px-3 border border-gray-300 dark:border-gray-200 rounded text-gray-600  dark:text-gray-400  appearance-none bg-transparent"
                  disabled=""
                >
                  <option
                    class="text-sm text-gray-600 dark:text-gray-400"
                  >
                    Status Reports
                  </option>
                  <option
                    class="text-sm text-gray-600 dark:text-gray-400"
                  >
                    Status Reports
                  </option>
                  <option
                    class="text-sm text-gray-600 dark:text-gray-400"
                  >
                    Status Reports
                  </option>
                  <option
                    class="text-sm text-gray-600 dark:text-gray-400"
                  >
                    Status Reports
                  </option>
                  <option
                    class="text-sm text-gray-600 dark:text-gray-400"
                  >
                    Status Reports
                  </option>
                </select>
              </div>
              <div
                class="relative w-full lg:w-1/4 my-2 lg:my-0 cursor-pointer lg:mx-2 xl:mx-4 z-0 border focus:border-gray-800 border-gray-300 dark:border-gray-200 rounded p-2"
              >
                <div
                  class="z-0 w-10 px-2 absolute inset-0 m-auto z-0 border-l mr-0 flex items-center text-slate-700"
                >
                  <svg
                    class="icon cursor-pointer icon-tabler icon-tabler-calendar-event"
                    fill="none"
                    height="20"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="1.5"
                    viewBox="0 0 24 24"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      stroke="none"
                    />
                    <rect
                      height="16"
                      rx="2"
                      width="16"
                      x="4"
                      y="5"
                    />
                    <line
                      x1="16"
                      x2="16"
                      y1="3"
                      y2="7"
                    />
                    <line
                      x1="8"
                      x2="8"
                      y1="3"
                      y2="7"
                    />
                    <line
                      x1="4"
                      x2="20"
                      y1="11"
                      y2="11"
                    />
                    <rect
                      height="2"
                      width="2"
                      x="8"
                      y="15"
                    />
                  </svg>
                  <input
                    class="absolute right-0 z-10 opacity-0 cursor-pointer"
                    type="date"
                  />
                </div>
                <p
                  class="text-sm text-gray-600 dark:text-gray-400"
                >
                  Schedule Dates
                </p>
              </div>
              <button
                class="focus:shadow-outline-gray border border-transparent w-auto lg:w-1/4 my-2 lg:my-0 lg:ml-2 xl:ml-4 bg-slate-700 transition focus:outline-none focus:border-gray-800 focus:shadow-outline-gray duration-150 ease-in-out hover:bg-slate-600 rounded text-white px-6 py-2 text-sm"
              >
                Add Sport Event
              </button>
            </div>
          </div>
          <div
            class="w-full overflow-x-scroll xl:overflow-x-hidden"
          >
            <div>
              No data available
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="py-12 min-h-screen"
    >
      <p
        class="text-left ml-8 text-base sm:text-lg md:text-xl lg:text-2xl font-bold leading-normal text-gray-800"
      >
        Sport events
      </p>
      <div
        class="mx-auto container bg-white dark:bg-gray-800 shadow rounded"
      >
        <div
          class="flex flex-col lg:flex-row p-8 justify-between items-start lg:items-stretch w-full"
        >
          <div
            class="w-full lg:w-1/4 xl:w-1/3 flex flex-col lg:flex-row items-start lg:items-center"
          >
            <div
              class="w-full relative mb-2 lg:mb-0 lg:mr-4"
            >
              <div
                class="absolute text-gray-600 dark:text-gray-400 flex items-center pl-4 h-full"
              >
                <svg
                  class="icon icon-tabler icon-tabler-search"
                  fill="none"
                  height="18"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="1.5"
                  viewBox="0 0 24 24"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M0 0h24v24H0z"
                    stroke="none"
                  />
                  <circle
                    cx="10"
                    cy="10"
                    r="7"
                  />
                  <line
                    x1="21"
                    x2="15"
                    y1="21"
                    y2="15"
                  />
                </svg>
              </div>
              <label
                class="hidden"
                for="search"
              />
              <input
                class="bg-gray-100 dark:bg-gray-700 text-gray-600 dark:text-gray-400 focus:outline-none focus:border focus:border-indigo-700 font-normal w-full h-10 flex items-center pl-12 text-sm border-gray-300 dark:border-gray-200 rounded border"
                id="search"
                placeholder="Search"
              />
            </div>
          </div>
          <div
            class="w-full lg:w-3/4 xl:w-2/3 flex flex-col lg:flex-row items-start lg:items-center justify-between"
          >
            <div
              class="relative w-full lg:w-1/4 my-2 lg:my-0 lg:mx-2 xl:mx-4 z-10"
            >
              <div
                class="absolute z-0 inset-0 m-auto mr-2 xl:mr-4 z-0 w-5 h-5"
              >
                <svg
                  class="icon cursor-pointer icon-tabler icon-tabler-chevron-down"
                  fill="none"
                  height="20"
                  stroke="#a0aec0"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="1.5"
                  viewBox="0 0 24 24"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M0 0h24v24H0z"
                    stroke="none"
                  />
                  <polyline
                    points="6 9 12 15 18 9"
                  />
                </svg>
              </div>
              <select
                aria-label="Selected tab"
                class="relative z-10 cursor-pointer focus:outline-none focus:border-gray-800 focus:shadow-outline-gray text-sm form-select block w-full py-2 px-2 xl:px-3 border border-gray-300 dark:border-gray-200 rounded text-gray-600 dark:text-gray-400 appearance-none bg-transparent"
                disabled=""
              >
                <option
                  class="text-sm text-gray-600 dark:text-gray-400"
                >
                  Representatives
                </option>
                <option
                  class="text-sm text-gray-600 dark:text-gray-400"
                >
                  Representatives
                </option>
                <option
                  class="text-sm text-gray-600 dark:text-gray-400"
                >
                  Representatives
                </option>
                <option
                  class="text-sm text-gray-600 dark:text-gray-400"
                >
                  Representatives
                </option>
                <option
                  class="text-sm text-gray-600 dark:text-gray-400"
                >
                  Representatives
                </option>
              </select>
            </div>
            <div
              class="relative w-full lg:w-1/4 my-2 lg:my-0 lg:mx-2 xl:mx-4 z-10"
            >
              <div
                class="z-0 absolute inset-0 m-auto mr-2 xl:mr-4 z-0 w-5 h-5"
              >
                <svg
                  class="icon cursor-pointer icon-tabler icon-tabler-chevron-down"
                  fill="none"
                  height="20"
                  stroke="#a0aec0"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="1.5"
                  viewBox="0 0 24 24"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M0 0h24v24H0z"
                    stroke="none"
                  />
                  <polyline
                    points="6 9 12 15 18 9"
                  />
                </svg>
              </div>
              <select
                aria-label="Selected tab"
                class="relative z-10 cursor-pointer focus:outline-none focus:border-gray-800 focus:shadow-outline-gray text-sm form-select block w-full py-2 px-2 xl:px-3 border border-gray-300 dark:border-gray-200 rounded text-gray-600  dark:text-gray-400  appearance-none bg-transparent"
                disabled=""
              >
                <option
                  class="text-sm text-gray-600 dark:text-gray-400"
                >
                  Status Reports
                </option>
                <option
                  class="text-sm text-gray-600 dark:text-gray-400"
                >
                  Status Reports
                </option>
                <option
                  class="text-sm text-gray-600 dark:text-gray-400"
                >
                  Status Reports
                </option>
                <option
                  class="text-sm text-gray-600 dark:text-gray-400"
                >
                  Status Reports
                </option>
                <option
                  class="text-sm text-gray-600 dark:text-gray-400"
                >
                  Status Reports
                </option>
              </select>
            </div>
            <div
              class="relative w-full lg:w-1/4 my-2 lg:my-0 cursor-pointer lg:mx-2 xl:mx-4 z-0 border focus:border-gray-800 border-gray-300 dark:border-gray-200 rounded p-2"
            >
              <div
                class="z-0 w-10 px-2 absolute inset-0 m-auto z-0 border-l mr-0 flex items-center text-slate-700"
              >
                <svg
                  class="icon cursor-pointer icon-tabler icon-tabler-calendar-event"
                  fill="none"
                  height="20"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="1.5"
                  viewBox="0 0 24 24"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M0 0h24v24H0z"
                    stroke="none"
                  />
                  <rect
                    height="16"
                    rx="2"
                    width="16"
                    x="4"
                    y="5"
                  />
                  <line
                    x1="16"
                    x2="16"
                    y1="3"
                    y2="7"
                  />
                  <line
                    x1="8"
                    x2="8"
                    y1="3"
                    y2="7"
                  />
                  <line
                    x1="4"
                    x2="20"
                    y1="11"
                    y2="11"
                  />
                  <rect
                    height="2"
                    width="2"
                    x="8"
                    y="15"
                  />
                </svg>
                <input
                  class="absolute right-0 z-10 opacity-0 cursor-pointer"
                  type="date"
                />
              </div>
              <p
                class="text-sm text-gray-600 dark:text-gray-400"
              >
                Schedule Dates
              </p>
            </div>
            <button
              class="focus:shadow-outline-gray border border-transparent w-auto lg:w-1/4 my-2 lg:my-0 lg:ml-2 xl:ml-4 bg-slate-700 transition focus:outline-none focus:border-gray-800 focus:shadow-outline-gray duration-150 ease-in-out hover:bg-slate-600 rounded text-white px-6 py-2 text-sm"
            >
              Add Sport Event
            </button>
          </div>
        </div>
        <div
          class="w-full overflow-x-scroll xl:overflow-x-hidden"
        >
          <div>
            No data available
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
